client/lib/users/actions.js

```diff
@@ -11,16 +11,18 @@ const Dispatcher = require( 'dispatcher' ),
 	UsersStore = require( 'lib/users/store' );
 
 const UsersActions = {
-	fetchUsers: fetchOptions => {
+	fetchUsers: ( fetchOptions, silentUpdate = false ) => {
 		const paginationData = UsersStore.getPaginationData( fetchOptions );
 		if ( paginationData.fetchingUsers ) {
 			return;
 		}
 		debug( 'fetchUsers', fetchOptions );
-		Dispatcher.handleViewAction( {
-			type: 'FETCHING_USERS',
-			fetchOptions: fetchOptions
-		} );
+		if ( ! silentUpdate ) {
```

Comment:
It dispatches the `FETCHING_USERS` event, which updates the UI. In the cases of people management and the author selector, that results in a placeholder being put at the bottom of the list.